addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -187,6 +202,18 @@ export class EmojiPicker extends Component {
         });
     }
 
+    get itemsAmount() {
```

Comment:
Maybe "numberOfItems" would be better English?

Reply:
"amount" if better than "number".
`itemsAmount` vs. `amountOfItems` is samey. 1st one is more readable IMO.